src/core/ledger/distributionSummary/allocationSummary.js

```diff
@@ -0,0 +1,68 @@
+// @flow
+
+import {Ledger} from "../ledger";
+import {type Distribution} from "../distribution";
+import {type Allocation} from "../grainAllocation";
+import {type GrainReceipt} from "../grainAllocation";
+import * as G from "../grain";
+import sortBy from "../../../util/sortBy";
+import {type IdentityId} from "../../identity";
+import * as NullUtil from "../../../util/null";
+import {type CurrencyDetails} from "../../../api/currencyConfig";
+import {formatCenter} from "./distributionSummary";
+
+export function allocationMarkdownSummary(
+  distribution: Distribution,
+  allocation: Allocation,
+  ledger: Ledger,
+  currencyDetails: CurrencyDetails
+) {
+  const {suffix: currencySuffix} = currencyDetails;
+  const {receipts} = allocation;
+  const totalDistributed: G.Grain = getTotalDistributed(receipts);
+  console.log(`## ${allocation.policy.policyType} Policy`);
```

Comment:
log the full policy, not just the type. would be nice to see the extra parameters, the budget, etc.

(simple: use json-stable-stringify. will display the budgets in raw string form, which is wonky. fancy: update the policies to have some better stringification support. i'm fine with the simple approach for now.)

Reply:
Done, added some simple `toString` functions for the policy types.